0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -124,34 +83,23 @@ The response MUST contain at least the following headers:
 | `Content-Type`  | MUST be `application/spsp4+json` to indicates the response is encoded as [JSON](http://www.json.org/) and that the ILP payment should be sent via STREAM. |
 | `Cache-Control` | Indicates how long the SPSP Client should cache the response. See supported cache-control directives below. |
 
-To handle as many transactions per second as possible, the SPSP Client caches results from the SPSP Server. The information returned by the SPSP Server is not expected to change rapidly, so repeated requests for the same information are usually redundant. The server communicates how long to cache results for using the HTTP-standard [`Cache-Control` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control) in the responses to RESTful API calls.
+To handle as many transactions per second as possible, the SPSP Client caches results from the SPSP Server. The information returned by the SPSP Server is not expected to change rapidly, so repeated requests for the same information are usually redundant. The SPSP Server communicates how long to cache results for using the HTTP-standard [`Cache-Control` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control) in the responses to RESTful API calls.
 
 The SPSP Client understands the following Cache-Control directives:
 
 | Directive     | Description                                                  |
 |:--------------|:-------------------------------------------------------------|
-| `max-age=<i>` | The client should cache this response for `<i>` seconds. `<i>` MUST be a positive integer |
-| `no-cache`    | The client must not cache this response |
+| `max-age=<i>` | The SPSP Client should cache this response for `<i>` seconds. `<i>` MUST be a positive integer. |
+| `no-cache`    | The SPSP Client must not cache this response. |
 
 ##### Response Body
 
 The response body is a JSON object that includes basic account details necessary for setting up payments.
```

Comment:
Should specify that more fields can be added, and implementations MUST ignore fields they don't understand. This way when we extend the protocol you won't have clients that throw errors because the server put in some kind of metadata that the client didn't understand.